docs/getting-started/cli/init.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Initialising new Subsembly project
+---
+
+# subsembly init
+
+### Overview
+
+Initialises new **`Subsemly`** project to the specified path. Path to the new **`Subsembly`** project should be an empty or non-existing directory.
+
+### Syntax
+
+```text
+subsembly init [to]
+```
+
+#### Parameters
+
+* **`--to`** - \[optional\] initialisation directory for the new **`Subsembly`** project
+
+### Examples
+
+```text
```

Comment:
Maybe we could show the created files in the directory after the execution of the command in some way.?